about.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/abtstyle.css">
+</head>
+<body>
+    <header>
+        <div class="upper_navbar">
+            <ul>
+             <li><a href="#"><img src="images/linkedinlogo.png" alt="Linked in nav item"></a></li>
+             <li><a href="#"><img src="images/twitterlogo.png" alt="Linked in nav item"></a></li>
+             <li><a href="#">English</a></li>
+             <li><a href="#">My Page</a></li>
+             <li><a href="#">Logout</a></li>          
+          </ul>
+        </div>
+        <nav>
+            <button type="button" id="hamburger-btn"><img src="images/hamburger.png" alt="Hamburger button"></button>
+            <div class="desktop-navbar">
+                <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                    <li><a href="#tstmons">Top Customers</a></li>
+                    <li><a href="#brands">Brands</a></li>
+                    <li><a href="#wrhs">Visit</a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="hero" id="hero">
+             <h3>Hello! Online Available Furniture</h3>
+             <h1>Beautifully Designed Furniture</h1>
+             <p class="msg">A website where all kind of furniture is available. Beatifully designed and best furniture is available and it's delivery is also available. 100%% best quality furniture is available at your door step.
+                 You can choose and buy your favourite furniture from our website and we can deliver it to you.
+             </p>
+            <p id="contact">
+               Please contact us via email for any further question about Homifyers online furniture shop
+               <a href="#">admin@homifyers.com</a>
+           </p>
+        </section>
+        <section class="middle">
+            <h2>Homifyers logo</h2>
+            <div class="sprtr"></div>
+            <p class="logo_txt">The logo of Homifyers was decided by its owner in 2021</p>
+            <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+            <h2>See old pictures of our warehouse</h2>
+            <div class="sprtr"></div>
+            <div class="old_pics">
+                <img src="images/old1.jpg" alt="first picture of warehouse">
+                <img src="images/old2.webp" alt="second picture of warehouse">
+            </div>
```

Comment:
- [ ] The two images on the about page should contain a text-centered over them, kindly implement this so your website looks like the provided design

|Your design|Provided design|
|-|-|
|![Screenshot_20221028_224550](https://user-images.githubusercontent.com/25227022/198738472-536dcd75-a730-4929-9696-4e961d0bd42a.png)|![Screenshot_20221028_224652](https://user-images.githubusercontent.com/25227022/198738485-2df8abcf-0e98-4acd-946a-3ab13781e879.png)|